qt/applications/workbench/workbench/plotting/plotscriptgenerator/lines.py

```diff
@@ -0,0 +1,96 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+#  This file is part of the mantid workbench.
+
+from __future__ import (absolute_import, unicode_literals)
+
+from matplotlib.container import ErrorbarContainer
+
+from mantid.plots.helperfunctions import errorbars_hidden
+from mantidqt.widgets.plotconfigdialog.curvestabwidget import CurveProperties, get_ax_from_curve
+from workbench.plotting.plotscriptgenerator.utils import convert_args_to_string, clean_variable_name
+
+BASE_CREATE_LINE_COMMAND = "plot({})"
+BASE_ERRORBAR_COMMAND = "errorbar({})"
+PLOT_KWARGS = [
+    'alpha', 'color', 'drawstyle', 'fillstyle', 'label', 'linestyle', 'linewidth', 'marker',
+    'markeredgecolor', 'markeredgewidth', 'markerfacecolor', 'markerfacecoloralt', 'markersize',
+    'markevery', 'solid_capstyle', 'solid_joinstyle', 'visible', 'zorder'
+]
+
+
+def generate_plot_command(artist):
```

Comment:
The final script is quite verbose with arguments that I'm not sure are required but could be defaults? I can see an argument that one it's simpler to code and two it shows users what the options are. Is it worth, especially for `plot` that we at least add some formatting, newlines and indentation, so that the user is not forced to scroll too far?